spring-integration-rmi/src/main/resources/org/springframework/integration/rmi/config/spring-integration-rmi-5.0.xsd

```diff
@@ -114,6 +114,20 @@
 							</xsd:documentation>
 						</xsd:annotation>
 					</xsd:attribute>
+					<xsd:attribute name="configurer" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								Specify a 'RmiProxyFactoryBeanConfigurer' to further configure the RmiProxyFactoryBean
+								before it creates the proxy - use this, for example,
+								to customize the RemoteInvocationFactory.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type type="org.springframework.integration.rmi.RmiProxyFactoryBeanConfigurer"/>
```

Comment:
Should be `org.springframework.integration.rmi.RmiOutboundGateway.RmiProxyFactoryBeanConfigurer`.

I hope... Because IDEA doesn't want to jump to that inner class fro XSD.